# 20090223: wsa: troll-ftpd is discontinued in favor of pure-ftpd. 1.28 was the last version.
diff -urN troll-ftpd-1.28/ls.c troll-ftpd-1.28-ptx1/ls.c
--- troll-ftpd-1.28/ls.c	2002-02-05 12:51:33.000000000 +0100
+++ troll-ftpd-1.28-ptx1/ls.c	2006-01-02 16:32:55.000000000 +0100
@@ -587,7 +587,7 @@
 		opt_t = 0;
 		break;
 	    default:
-
+		break;
 	    }
 	}
 	while ( isspace( *arg ) )
